doc/release-notes-31679.md

```diff
@@ -0,0 +1,11 @@
+Install changes
+---------------
+
+The `test_bitcoin`, `test_bitcoin-qt`, `bench_bitcoin`, `bitcoin-chainstate`,
+`bitcoin-node`, and `bitcoin-wallet` binaries are now installed in
+`$PREFIX/libexec` rather than `$PREFIX/bin`. If you are using a binary release
+or building from source with default build options, most of these programs are
+not built by default anyway so these changes may not be noticeable.
```

Comment:
04e51ba1110e13598e0b11a35ce5abf4a1789f53: now that the wrapper binary is available, the release note could enumerate how to call these now: `bitcoin test`, etc.